### DISCUSSION TITLE
Jobs debugging

### DISCUSSION THREAD

**Discussion by ilsenem** [roadrunner-server/roadrunner#1285] [Q&A] [answered]
HTTP debugging is well covered in RR documentation and even have dedicated configuration option in `.rr.yaml`. For Jobs there is not such helpful information and debugging process is not so intuitive.

Is there an option to add something like `jobs.pool.debug` to force RR limit number of jobs wokers to 1 and restart worker on every task recieved?

**Comment by rustatian** [comment, marked as answer]
Thanks, @ilsenem; the PR was merged: https://github.com/spiral/roadrunner-docs/pull/113
